From 4805d747cdab8044eaff90e264a62fbd3efba964 Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Tue, 2 Apr 2019 09:53:31 +0300
Subject: [PATCH 131/386] arm64: dts: cn9130: change pinctrl compatible string

Change pinctrl compatible string to cp115-standalone-pinctrl,
which allows to configure any MPP's to any possible function.

Change-Id: I41d08ef505e554e1dd9f8831dd2302f80e9e973b
Signed-off-by: Alex Leibovich <alexl@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/6732
Reviewed-by: Grzegorz Jaszczyk <Grzegorz.Jaszczyk@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130.dtsi b/arch/arm64/boot/dts/marvell/cn9130.dtsi
index b78b4f482063..25fed7b291d2 100644
--- a/arch/arm64/boot/dts/marvell/cn9130.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130.dtsi
@@ -62,7 +62,7 @@
 
 &cp0_syscon0 {
 	cp0_pinctrl: pinctrl {
-		compatible = "marvell,armada-8k-cpm-pinctrl";
+		compatible = "marvell,cp115-standalone-pinctrl";
 
 		cp0_i2c0_pins: cp0-i2c-pins-0 {
 			marvell,pins = "mpp37", "mpp38";
-- 
2.17.1

